lib/pages/settings_screen.dart

```diff
@@ -47,9 +50,93 @@ class _SettingsScreenState extends State<SettingsScreen> {
           },
         ),
       ),
-      body: const Center(
-        child: Text('Settings'),
-    ),
+      body: Stack(
+        children: [
+          Container(
+            padding: const EdgeInsets.all(16),
+            color: const Color(0xFF1A1A1A), // Set background color to black
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: [
+                Row(
+                  children: [
+                    SizedBox(
+                      width: 100,//size of profile image
+                      height: 100,//size of profile image
+                      child: CachedNetworkImage(
+                        placeholder: (context, url) => const CircularProgressIndicator(),
+                        imageUrl: 'https://picsum.photos/250?image=9',
+                      ),
+                    ),
+                    const SizedBox(width: 40), // The gap between the image and username
+                    Column(
+                      crossAxisAlignment: CrossAxisAlignment.start,
+                      children: [
+                        const Text(
+                          'Xxx', // Username text
+                          style: TextStyle(fontSize: 30, color: Colors.white),
+                        ),
+                        const SizedBox(height: 12), // Gap between the username and "Edit Profile" button
+                        Container(
+                          width: 150, // Set the width of the button
+                          decoration: BoxDecoration(
+                            color: const Color(0xFFE1F0CF), // Set the background color to green
+                            borderRadius: BorderRadius.circular(100), // Rounded corners
+                          ),
+                          child: TextButton(
+                            onPressed: () {
+                              // Navigator.push(
+                              //   context,
+                              //   MaterialPageRoute(builder: (context) => const FirstRoute()),
+                              // );
+                            },
+                            child: const Text(
+                              'Edit Profile',
+                              style: TextStyle(color: Color(0xFF1A1A1A),fontSize: 16,), // Change the button text color to black
+                            ),
+                          ),
+                        ),
+                      ],
+                    ),
+                  ],
+                ),
+                const SizedBox(height: 30),
+                const Divider(),
+                const SizedBox(height: 10),
+                ProfileMenuItem(
```

Comment:
Could move the words "Notifications", "Help Centre"... etc slightly more to the left, nearer to their icons
And set their font weight to bold